src/anti_patterns/deny-warnings.md

```diff
@@ -106,6 +106,6 @@ certain that there will be more deprecated APIs in the future.
   `rustc --help` for a general list of options
 - [rust-clippy] is a collection of lints for better Rust code
 
-[rust-clippy]: https://github.com/Manishearth/rust-clippy
+[rust-clippy]: https://github.com/rust-lang/rust-clippy
```

Comment:
The link was to a fork of the official rust-clippy repository. I don't see anything interesting in that fork. Let's link to the original.